zettarepl/replication/run.py

```diff
@@ -414,6 +414,15 @@ def resume_replications(step_templates: [ReplicationStepTemplate], observer=None
                     context.snapshots_total_by_replication_step_template[step_template] = 1
                     resumed = True
 
+                    # ZFS does not automatically mount after resuming with receive_resume_token,
+                    # let's do it ourselves
+                    for dst_dataset in sorted(step_template.dst_context.datasets.keys(), key=len):
```

Comment:
Dont we have to check the mountpoint property?

If its not "legacy" or we have overridden that prop?